### DISCUSSION TITLE
site unicorn off ?

### DISCUSSION THREAD

**Discussion by Rip4568** [django-commons/django-unicorn#489] [General]
I was trying to access the site to check the documentation and nothing appears, just an error message saying: {Nothing here yet :/ }

@adamghill

**Comment by adamghill** [comment]
Sorry, it should be back now!

**Reply by Rip4568** [reply]
Thanks for responding you are an amazing person!
I am having trouble making the child components work, could you give me an example template ?

**Reply by adamghill** [reply]
Child components are especially finicky. 

I try to keep a sample of every feature in the `example` project. For child components, the templates are here: https://github.com/adamghill/django-unicorn/tree/main/example/unicorn/templates/unicorn/nested. The views are in https://github.com/adamghill/django-unicorn/tree/main/example/unicorn/components/nested. `table` is the parent and `row` is the child. Hopefully that helps!
